public-dashboard-client/src/page-sentencing/PageSentencing.js

```diff
@@ -46,13 +47,25 @@ export default function PageSentencing() {
         : getCohortOptions(apiData.recidivism_rates_by_cohort_by_year),
     [apiData.recidivism_rates_by_cohort_by_year, isLoading]
   );
-  const [selectedCohorts, setSelectedCohorts] = useState();
+  const [selectedCohorts, setSelectedCohorts] = useState([]);
   const [highlightedCohort, setHighlightedCohort] = useState();
+  const [recidivismDimension, setRecidivismDimension] = useState(
```

Comment:
Agreed offline that these charts should be on the Prison page instaed

Reply:
yes I am going to deal with that in a separate PR. Thanks for catching it!